src/ensembl/src/content/app/custom-download/containers/content/attributes-accordion/sections/orthologues/Orthologues.tsx

```diff
@@ -0,0 +1,235 @@
+import React, { useCallback } from 'react';
+import { connect } from 'react-redux';
+import { RootState } from 'src/store';
+import CheckBoxGrid, {
+  getAttributesCount
+} from 'src/content/app/custom-download/components/checkbox-grid/CheckboxGrid';
+
+import {
+  getOrthologueAttributes,
+  getOrthologueSearchTerm,
+  getOrthologueSpecies,
+  getOrthologueShowBestMatches,
+  getOrthologueShowAll,
+  getOrthologueApplyToAllSpecies
+} from '../../state/attributesAccordionSelector';
+
+import {
+  setOrthologueAttributes,
+  setOrthologueSearchTerm,
+  fetchOrthologueSpecies,
+  setOrthologueSpecies,
+  setOrthologueShowBestMatches,
+  setOrthologueShowAll,
+  setOrthologueApplyToAllSpecies
+} from '../../state/attributesAccordionActions';
+
+import Input from 'src/shared/input/Input';
+
+import styles from './Orthologues.scss';
+
+import { orthologueAttributes } from 'src/content/app/custom-download/sampledata.tsx';
+
+type ownProps = {
+  hideUnchecked?: boolean;
+  hideTitles?: boolean;
+};
+
+type Props = ownProps & StateProps & DispatchProps;
+
+const Orthologue = (props: Props) => {
+  const attributesOnChangeHandler = useCallback(
+    (status: boolean, subSection: string, attributeId: string) => {
+      const newOrthologueAttributes = { ...props.orthologueAttributes };
+
+      newOrthologueAttributes[subSection][attributeId].checkedStatus = status;
+
+      props.setOrthologueAttributes(newOrthologueAttributes);
+    },
+    [props.orthologueAttributes]
+  );
```

Comment:
Here and elsewhere: what is the purpose of using the `useCallback` hook? According to the docs, it returns a memoized function, which would have been handy if we did some heavy computation inside this function, but we are not; it is used for the sole purpose of calling a callback passed through props. Why not just have a regular function then, instead of the one wrapped in `useCallback`?

Reply:
Removed all the unnecessary useCallbacks.